front/lib/resources/data_source_view_resource.ts

```diff
@@ -157,6 +164,32 @@ export class DataSourceViewResource extends ResourceWithVault<DataSourceViewMode
     return DataSourceResource.fetchByModelIdWithAuth(auth, this.dataSourceId);
   }
 
+  // Updating.
+  async updateParents(
+    auth: Authenticator,
+    parentsIn: string[] | null
+  ): Promise<Result<undefined, Error>> {
+    try {
```

Comment:
Why do we need the try/catch here?

Reply:
not needed actually, we wil only catch sequelize errors